src/legacy/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -87,7 +87,14 @@ const coreSystem = new CoreSystem({
       buildNum: 1234,
       devMode: true,
       uiSettings: {
-        defaults: ${JSON.stringify(defaultUiSettings, null, 2)
+        defaults: ${JSON.stringify(
+          defaultUiSettings,
+          (key, value) => {
+            if (value instanceof Type) return null;
```

Comment:
That's a hack to fix karma tests. 
We might have to refactor karma tests when move ui settings default declaration to the NP.